content/admin.py

```diff
@@ -7,6 +7,15 @@ class EduProgramAdmin(admin.ModelAdmin):
     list_display = ('title','category', 'order')
 
 
-admin.site.register(OurStory)
-admin.site.register(OurTeam)
+class OurStoryAdmin(admin.ModelAdmin):
+    fields = ['id','order','text']
+    list_display=('id', 'order', 'text')
+
+
+class OurTeamAdmin(admin.ModelAdmin):
+    fields = ['id', 'text']
```

Comment:
@hassanabidpk  I probably have copied from the documentation. I will delete both.